daemon-testcases/SymlinkTest.cs

```diff
@@ -27,6 +52,7 @@ public void WindowsSymlinkExample()
                     Assert.Fail();
                 }
             }
+
             Assert.Pass("Linux Detected - Asserting as true for this test case.");
```

Comment:
Don't do this, use this instead: https://github.com/nunit/docs/wiki/Platform-Attribute to target specific platforms/runtimes.

Reply:
This seems out of date, and also poorly documented.
The first thing I came across while attempting to try this is the [Platform()] attribute doesn't appear to exist in the current version of NUnit we have installed.
![image](https://user-images.githubusercontent.com/33913289/44340323-7a3ea700-a452-11e8-97cd-75eb5454f808.png)

Secondly, there doesn't appear to be a specifier for Windows 10, nor does there appear to be an generic mention of all windows platforms. 

There's a few I would guess, but even if it worked it would be taking random shots in the dark at some of the following: Win, Win32, Win32Windows, etc.